src/main/java/org/jabref/logic/importer/fileformat/PdfContentImporter.java

```diff
@@ -437,7 +514,8 @@ Optional<BibEntry> getEntryFromPDFContent(String firstpageContents, String lineS
             entry.setField(StandardField.KEYWORDS, keywords);
         }
         if (title != null) {
-            entry.setField(StandardField.TITLE, title);
+//            title = guessBetterTitleInMetaData(buildMetaData(author, editor, abstractT, keywords, title, conference, doi, series, volume, number, pages, year, publisher));
+            entry.setField(StandardField.TITLE, titleByPosition == null || titleByPosition.isEmpty() ? title : titleByPosition);
```

Comment:
No, this is flawed

- If there is no title - the titleByPosition should be used.
- Use `org.jabref.model.strings.StringUtil#isNullOrEmpty` for clear checking
- Split up `?`in real `if/else`
- Do a check for `!StringUtil.isNullOrEmpty` first, then store this in `title` field.
- Then, the if `title != null` and setting the field can be kept. ( ` entry.setField(StandardField.TITLE, title`)